src/ray/core_worker/experimental_mutable_object_manager.h

```diff
@@ -30,23 +30,80 @@
 #include "src/ray/protobuf/common.pb.h"
 
 namespace ray {
+namespace experimental {
 
-class ExperimentalMutableObjectManager {
+class MutableObjectManager {
```

Comment:
Are the changes to this class mainly code cleanups / self-contained? If so, could you extract them into a separate PR?

Reply:
Done. #44371